include/xrpl/protocol/Asset.h

```diff
@@ -157,6 +167,21 @@ operator!=(Asset const& lhs, Asset const& rhs)
     return !(lhs == rhs);
 }
 
+constexpr bool
+operator<(Asset const& lhs, Asset const& rhs)
```

Comment:
Can you pls replace this with `operator<=>` ? It could look like this

```c++
    friend constexpr auto
    operator<=>(Asset const& lhs, Asset const& rhs)
    {
        return std::visit(
            [&]<typename TLhs, typename TRhs>(
                TLhs const& issLhs, TRhs const& issRhs) {
                if constexpr (std::is_same_v<TLhs, TRhs>)
                    return std::weak_ordering(issLhs <=> issRhs);
                else
                    return std::weak_ordering::greater;
            },
            lhs.issue_,
            rhs.issue_);
    }
```

Reply:
It would be great if we could give `std::strong_ordering` to `operator<=>(Issue const& lhs, Issue const& rhs)` but I wouldn't want that change in this PR even if it is correct one (and I am not assuming that it is)

Reply:
Sure, will do. I actually added it in MPT-V2 (still a long way from release).